### PR TITLE
[Edge 1018 Fix-1] Edge 1018.0.1 Fix release updates

### PR DESCRIPTION
**Pull request by bbyreddy**

### CODE REVIEW

File: content/edge-kubernetes/k8-using-edge-bundle/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+weight: 10
+title: Getting started
+layout: redirect
+---
+
+Read about the concepts behind the Cumulocity IoT platform. Get familiar with the platform UI or directly get started by connecting your phone.
```
Comment (MWindel) [inline content/edge-kubernetes/k8-using-edge-bundle/getting-started.md:7]: add shortcode for Cumulocity IoT

**Comment by bbyreddy** [comment]
Hi @MWindel and @BeateRixen , please review this PR for 10.18.0.1 (Fix-1) of Edge on K8s version. Thanks!

**Review by MWindel — APPROVED** [review]

File: content/edge-kubernetes/installing-edge-on-k8-bundle/install-operator.md [hunk 2]
```diff
@@ -4,32 +4,38 @@ title: Installing the Edge Operator
...
+To request the Edge registry credentials, contact the {{< company-sag >}} logistics team for your region:
 * North and South America: LogisSrvus@softwareagusa.com
-* All Other Regions: LogisticsServiceCenterGER@softwareag.com {{< /c8y-admon-info >}}
+* All Other Regions: LogisticsServiceCenterGER@softwareag.com
+{{< /c8y-admon-info >}}
 
 By default, the Edge Operator is deployed within the **c8yedge** namespace. If you wish to install the Edge Operator and Edge in a different namespace, you can specify it using the `-n` option in the installation script.
 
-Run the following command to follow the logs for the Edge Operator pod:
+Run the following command to follow the logs for the Edge Operator pod, :
```
Comment (nickponomar) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/install-operator.md:35]: nit: comma before colon:
``` suggested
Run the following command to follow the logs for the Edge Operator pod:
```

**Review by nickponomar — APPROVED** [review]
Looks good to me

File: content/edge-kubernetes/installing-edge-on-k8-bundle/install-operator-offline.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
+---
+weight: 17
+title: Installing the Edge Operator (offline)
```
Comment (BeateRixen) [inline content/edge-kubernetes/installing-edge-on-k8-bundle/install-operator-offline.md:3]: Please unify the spelling to be in sync with the release notes PR where it is "Edge operator"

**Pull request merged**